scripts/deploy/iks/deploy-tekton.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash
+#
+# Copyright 2021 kubeflow.org
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Remove the x if you need no print out of each command
+set -ex
+
+# Need the following env
+# - PIPELINE_KUBERNETES_CLUSTER_NAME:       kube cluster name
+# - TEKTON_VERSION:                         tekton version
+# - TEKTON_NS:                              tekton namespace, defulat: tekton-pipeline
+
+MAX_RETRIES="${MAX_RETRIES:-5}"
+SLEEP_TIME="${SLEEP_TIME:-10}"
+EXIT_CODE=0
+
+TEKTON_NS="${TEKTON_NS:-"tekton-pipelines"}"
+# Previous versions use form: "previous/vX.Y.Z"
+TEKTON_VERSION="${TEKTON_VERSION:-"latest"}"
+TEKTON_MANIFEST="${TEKTON_MANIFEST:-https://storage.googleapis.com/tekton-releases/pipeline/${TEKTON_VERSION}/release.yaml}"
+TEKTON_MANIFEST_FILENAME=tekton-manifest.yaml
```

Comment:
May want to use environment/default pattern.

Reply:
sure and done